src/chocolatey.tests.integration/context/dependencies/hasdependency/2.0.1/hasdependency.nuspec

```diff
@@ -13,7 +13,7 @@
     <copyright />
     <tags>hasdependency admin</tags>
     <dependencies>
-      <dependency id="isdependency" />
+      <dependency id="isdependency" version="2.1.0" />

```

Comment:
This doesn't look right, it should depend on the earliest version to make it equivalent to the current dependency range. I believe the earliest version is `1.0.0` of this package.